### PR TITLE
#87

### PR DESCRIPTION
**Pull request by mensfeld**
@agwozdowski  @pavlo-vavruk  please review and merge if ok :)

### CODE REVIEW

File: lib/karafka/server.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      def bind_on_sigint
+        process.on_sigint do
+          Karafka::App.stop!
+          exit
+        end
+      end
+
+      # What should happen when we decide to quit with sigquit
+      def bind_on_sigquit
+        process.on_sigquit do
```
Comment (pavlo-vavruk) [inline lib/karafka/server.rb]: @mensfeld maybe, to get rid of duplication - move it to separate private method
```rb
def stop_app
  lambda do
    Karafka::App.stop!
    exit
  end
end
```
and use it 

```rb
def bind_on_sigint
  process.on_sigint(&stop_app)
end

def bind_on_sigquit
  process.on_sigquit(&stop_app)
end
```

Or is it redundant, how do you think?

Comment (mensfeld) [inline lib/karafka/server.rb]: it is redundant on purpose. If we would add more siqnal handlers than each of them would be separated

**Comment by pavlo-vavruk** [comment]
👍

**Comment by mensfeld** [comment]
thank you @agwozdowski  :)

**Pull request merged**